tutorials/axios/axios-patch.md

```diff
@@ -0,0 +1,13 @@
+The easiest way to make a PATCH request with [Axios](https://www.npmjs.com/package/axios) is the axios.patch() function. The first parameter is the url to which the request will be made, and the second parameter is the data you will be sending to change. You can include the other data that will not be changed as this will have no effect on the data stored. You can make a Patch request with axios as follows:
```

Comment:
I'm not sure what "You can include the other data that will not be changed as this will have no effect on the data stored" means?